examples/http-proxy/Cargo.toml

```diff
@@ -9,6 +9,6 @@ axum = { path = "../../axum" }
 hyper = { version = "1", features = ["full"] }
 hyper-util = "0.1.1"
 tokio = { version = "1.0", features = ["full"] }
-tower = { version = "0.4", features = ["make"] }
+tower = { version = "0.4", features = ["make", "util"] }
```

Comment:
This seems like an accidental change. The version did not change and I don't expect the feature to do anything on its own.

It's just an example though so I guess it doesn't hurt anything, feel free to ignore.

Reply:
Yeah I don't know exactly what happened here but it's definitely required, and should have been included before AFAICT.